execution/executor/Cargo.toml

```diff
@@ -38,7 +38,7 @@ config-builder = { path = "../../config/config-builder", version = "0.1.0" }
 executor-utils = { path = "../executor-utils", version = "0.1.0" }
 libra-config = { path = "../../config", version = "0.1.0" }
 libra-temppath = { path = "../../common/temppath", version = "0.1.0" }
-libradb = { path = "../../storage/libradb", version = "0.1.0" }
+libradb = { path = "../../storage/libradb", version = "0.1.0", features = ["fuzzing"] }
```

Comment:
Would this be added to line 48 instead? I think you are enabling fuzzing by default here?

Reply:
This is the dev-dependence, I AM enabling fuzzing by default (to get the hidden LibraDB::new_for_test).

adding it to [features] requires LibraDB to be in the main [dependencies] (although it can be "optinoal = true".